src/CallableHandler.php

```diff
@@ -122,4 +75,13 @@ public static function execute($callable, array $arguments = [])
             throw $exception;
         }
     }
+
+    protected static function getResolver()
```

Comment:
I didn't want to remove anything without discussion, but I agree.

> I agree with you about instantiate the resolver in the middleware, but this could be done in the process method, to avoid create unnecesary instances.

Okay, but I would still do some lazy initialization. Not really common, but one could hit the pipeline multiple times which would mean multiple initializations (which is what you wanted to avoid in the first place.)

But this is another story then. I will remove the resolve from the CallableHandler and consider this PR done.